rust/common/error/mod.rs

```diff
@@ -117,7 +117,7 @@ error_messages! { TypeQLError
     InsertModifiersRequireMatch { insert: String } =
         20: "The insert query '{insert}' contains query modifiers that require a 'match' clause be specified",
     VariableNotNamed =
-        21: "Anonymous variable encountered in a match query filter.",
+        21: "Anonymous variable encountered in a match query select.",
```

Comment:
Also we should use the new terminology, such as `select operator` - there is no 'match query' anymore :)